inc/actions.php

```diff
@@ -113,6 +113,7 @@ function uploadcare_attach($file) {
 
     if (get_option('uploadcare_download_to_server')) {
         $attached_file = uploadcare_download($file);
+        add_post_meta($attachment_id, '_is_local_file', true, true);
```

Comment:
Not sure about convention, but you're right.
I will fix it to be in a same manner as others.